ztp/gitops-subscriptions/argocd/example/acmpolicygenerator/README.md

```diff
@@ -1,5 +1,78 @@
-## ACM PolicyGenerator
-The [ACM PolicyGenerator](https://github.com/stolostron/policy-generator-plugin) examples under [here](https://github.com/openshift-kni/cnf-features-deploy/tree/master/ztp/gitops-subscriptions/argocd/example/acmpolicygenerator) are defining the DU profile policies using [ACM PolicyGenerator reference API](https://github.com/stolostron/policy-generator-plugin/blob/main/docs/policygenerator-reference.yaml). These examples will generate ACM policies same as the DU profile policies generated from [policygentemplates example](https://github.com/openshift-kni/cnf-features-deploy/tree/master/ztp/gitops-subscriptions/argocd/example/policygentemplates) AND specifically the policy content object-definition (source-crs) are identical.
+# ACM PolicyGenerator 
+The ACM Policy Generator Custom Resource (CRs) is the official template mechanism used to generate ACM policies. It is replacing Policy Generator Templates or PGT.
```

Comment:
Maybe something along the lines of 
``It is the recommended alternative to Policy Generator Templates (PGT).`` ?

Reply:
fixed